app/services/shopify/association_saver.rb

```diff
@@ -11,16 +11,14 @@ def save_pos_variant_id_for_variants(spree_variants, shopify_variants)
       end
 
       def save_pos_variant_id(spree_variant, shopify_variant)
-        spree_variant.pos_variant_id = shopify_variant.id
-        spree_variant.save
+        spree_variant.update_columns(pos_variant_id: shopify_variant.id)
```

Comment:
I know there was a question about how this doesn't initiate callbacks this morning. However, I read these methods like:

`we are updating the column directly in the db and bypassing the model completely`

This is why I think it reads fine and clues the reader into what's 'sort of' going on.